contracts/core/foundry.toml

```diff
@@ -20,7 +20,7 @@ fs_permissions = [
 ]
 
 allow_paths = ["../avs"]
-
+gas_limit = 9223372036854775807 # prev foundry default
```

Comment:
Recent foundry nightly version changed gas limit it seems. Tests were failing

Reply:
why were the tests failing / why is it okay to not use the new default?

Reply:
Tests were failing because they ran out of gas - not sure what they changed it too in new nightly release, it's not in docs. might be a bug, or they just decreased defaults